tests/test_iet.py

```diff
@@ -273,6 +276,59 @@ def _C_free(self):
 }"""
 
 
+def test_cpp_local_object():
+    """
+    Test C++ support for LocalObjects.
+    """
+
+    class MyObject(LocalObject):
+        dtype = CustomDtype('dummy')
+
+    # Locally-scoped objects are declared in the function body
+    lo0 = MyObject('obj0')
+
+    # Globally-scoped objects must not be declared in the function body
+    lo1 = MyObject('obj1', is_global=True)
+
+    # A LocalObject using both a template and a modifier
+    class SpecialObject(LocalObject):
+        dtype = CustomDtype('bar', ('int', 'float'), '&')
```

Comment:
`template=('int', 'float'), modifier='&'` since this is the "example" a bit better to be explicit for new users/...

Reply:
ok